app/post.go

```diff
@@ -281,7 +282,7 @@ func (a *App) CreatePost(post *model.Post, channel *model.Channel, triggerWebhoo
 
 	if pluginsEnvironment := a.GetPluginsEnvironment(); pluginsEnvironment != nil {
 		var rejectionError *model.AppError
-		pluginContext := a.PluginContext()
+		pluginContext := pluginContext(c)
```

Comment:
Wondering if it's possible to have this as a method for `request.Context` itself? Like `c.PluginContext`. Or will it introduce  cyclic dependencies?

Reply:
I think it won't create cyclic dependencies, I just wanted to decouple `app/request` from every package as it maybe could be used somewhere else.

Reply:
Ok